File: Ex7_IntentLogin/app/src/main/java/com/example/nt21_quiz/MainActivity.java
package com.example.nt21_quiz;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        //Tìm điều khiển
        Button btnDangNhap = findViewById(R.id.btnLogin);
        //Sự kiện
        btnDangNhap.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                //Lệnh xử lý ở đây
                Intent iLogin = new Intent(MainActivity.this,LoginActivity.class);
                startActivity(iLogin);
            }
        });
    }
}

File: Thigk2.NguyenDinhNguyen/app/src/main/java/com/example/thigk/MainActivity.kt
package com.example.thigk

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        val btnCau1 = findViewById<Button>(R.id.btn1)
        val btnCau2 = findViewById<Button>(R.id.btn2)
        val btnCau3 = findViewById<Button>(R.id.btn3)
        val btnCau4 = findViewById<Button>(R.id.btn4)

        btnCau1.setOnClickListener {
            startActivity(Intent(this, ActivityChucnang2::class.java))
        }
        btnCau2.setOnClickListener {
            startActivity(Intent(this, ActivityChucnang3::class.java))
        }
        btnCau3.setOnClickListener {
            startActivity(Intent(this, ActivityChucnang4::class.java))
        }
        btnCau4.setOnClickListener {
            startActivity(Intent(this, ActivityAboutMe::class.java))
        }
    }
}
